src/manage.c

```diff
@@ -7916,17 +7916,18 @@ delete_resource (const char *type, const char *resource_id, int ultimate)
  * @brief Union type for values of semctl actions
  */
 union semun {
-    int              val;    /* Value for SETVAL */
-    struct semid_ds *buf;    /* Buffer for IPC_STAT, IPC_SET */
-    unsigned short  *array;  /* Array for GETALL, SETALL */
-    struct seminfo  *__buf;  /* Buffer for IPC_INFO
-                                (Linux-specific) */
+    int              val;    ///< Value for SETVAL
```

Comment:
This is still indenting by 4 whereas all our `typedef struct`s and functions etc indent by 2.

I can understand breaking the convention to line up the field names though, if you like those.